src/NServiceBus.Core/Unicast/MessageHandlerRegistry.cs

```diff
@@ -13,11 +13,6 @@
     /// </summary>
     public class MessageHandlerRegistry
     {
-        internal MessageHandlerRegistry(Conventions conventions)
```

Comment:
Need to keep an internal constructor around to prevent the default public constructor from being added.

Reply:
Not sure why we made the ctor internal back in the day. Do we see any drawback in exposing the public ctor?

Reply:
I don't suppose there is any problem with allowing this. You'll need to update the API approval test, then.